### PR TITLE
Updated cgal to 4.9.1.

### PR DESCRIPTION
**Pull request by rmarsa**

### CODE REVIEW

File: src/cgal.mk [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # This file is part of MXE. See LICENSE.md for licensing information.
 
 PKG             := cgal
-$(PKG)_WEBSITE  := https://www.cgal.org/
+$(PKG)_WEBSITE  := http://www.cgal.org/
```
Comment (starius) [inline src/cgal.mk]: Why https URL was replaced with http?

**Comment by starius** [comment]
Tested on all 4 targets.

File: src/cgal.mk (hunk 1 above)
Comment (rmarsa) [inline src/cgal.mk]: No reason.  I fixed and pushed the change.

**Comment by starius** [comment]
Thanks!

**Pull request merged**